releasenotes/notes/go118-b3ac691999dd0015.yaml

```diff
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+security:
+  - |
+    The Agent is built with go1.18.6.
```

Comment:
Should we include the extra comment here from the other PR around the mem tweaking here as well?